DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/SelfAssessmentOverview.cshtml

```diff
@@ -64,10 +64,25 @@
                          view-data="@(new ViewDataDictionary(ViewData) { { "IsSupervisorResultsReviewed", Model.SelfAssessment.IsSupervisorResultsReviewed } })" />
             </div>
         }
+        <partial name="_OverviewActionButtons.cshtml" model=Model />
+        @if (competencyQuestionsSummary == competencyVerifiedSummary)

```

Comment:
@rshrirohit, we should be showing this if the number of optional competencies is less than the minimum number of optional competencies. This switch doesn't seem to be doing that?